src/main/java/uk/gov/hmcts/darts/common/entity/TranscriptionStatusEntity.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.darts.common.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.SequenceGenerator;
+import jakarta.persistence.Table;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Table(name = TranscriptionStatusEntity.TABLE_NAME)
+@Getter
+@Setter
+@EqualsAndHashCode(callSuper = false)
+public class TranscriptionStatusEntity {
+
+    public static final String TABLE_NAME = "transcription_status";
+    public static final String TRANSACTION_STATUS_ID = "trs_id";
```

Comment:
Typo on these globals - should be `TRANSCRIPTION_`

I'd prefer the new Transcription* entities to be in similar style to `TranscriptionEntity` - we don't need these columns exposed here if we are using jpa queries - I just find it harder to read the entity class.

This other style came about on the back of the Notification entity